wb/main/console_tool_wrapper/huggingface_model_downloader/tool.py

```diff
@@ -13,11 +13,23 @@
  You may obtain a copy of the License at
       https://software.intel.com/content/dam/develop/external/us/en/documents/intel-openvino-license-agreements.pdf
 """
+
+import re
 from pathlib import Path
 
+from wb.error.job_error import TransformersONNXConversionError
 from wb.main.console_tool_wrapper.console_parameter_validator import ConsoleParametersTypes
 from wb.main.console_tool_wrapper.python_console_tool import PythonModuleTool
-from wb.main.jobs.tools_runner.console_output_parser import ConsoleToolOutputParser
+from wb.main.jobs.interfaces.job_state import JobStateSubject
+from wb.main.jobs.tools_runner.console_output_parser import ConsoleToolOutputParser, skip_empty_line_decorator
+
+
+DOWNLOAD_PROGRESS_STRING_START = "Downloading:"
```

Comment:
are double quotes passing linting? @artyomtugaryov did we disabled single quote rule?

Reply:
Fix.